sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/CompatibilityMatrix.java

```diff
@@ -108,13 +114,11 @@ public static class Plugin {
     private final Map<String, String> compatibleVersionBySqVersion = new HashMap<>();
 
     private final boolean isSupportedBySonarSource;
-    private final boolean isBundled;
 
-    public Plugin(String name, String homepageUrl, boolean isSupportedBySonarSource, boolean isBundled) {
+    public Plugin(String name, String homepageUrl, boolean isSupportedBySonarSource) {
       this.name = name;
       this.homepageUrl = homepageUrl;
       this.isSupportedBySonarSource = isSupportedBySonarSource;
-      this.isBundled = isBundled;
     }
```

Comment:
the method isArchived can be dropped (sorry for the wrong place, but github does not allow me to put a comment there)

Reply:
done 👍